platform_versioned_docs/version-24.1/getting-started/proteinfold.mdx

```diff
@@ -0,0 +1,481 @@
+---
+title: "Protein structure prediction"
+description: "An introduction to running nf-core/proteinfold in Seqera Platform"
+date: "21 Jul 2024"
+tags: [platform, seqera pipelines, data studios, proteinfold, alphafold, colabfold, compute environment, aws]
+toc_max_heading_level: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This guide details how to perform best-practice analysis for protein 3D structure prediction on an AWS Batch compute environment in Platform. It includes: 
+
+- Creating AWS Batch compute environments to run your pipeline and downstream analysis
+- Adding the nf-core/proteinfold pipeline to your workspace 
+- Importing your pipeline input data
+- Launching the pipeline and monitoring execution from your workspace
+- Setting up a custom analysis environment with Data Studios
+
+:::info[**Prerequisites**]
+You will need the following to get started:
+
+- [Admin](../orgs-and-teams/roles.mdx) permissions in an existing organization workspace. See [Set up your workspace](./workspace-setup.mdx) to create an organization and workspace from scratch.
+- An existing AWS cloud account with access to the AWS Batch service.
+- Existing access credentials with permissions to create and manage resources in your AWS account. See [IAM](../compute-envs/aws-batch.mdx#iam) for guidance to set up IAM permissions for Platform. 
+:::
+
+## Compute environment
+
+The compute and storage requirements for protein structure prediction depend on the number and length of protein sequences being analyzed and the size of the database used for prediction by the deep learning models (AlphaFold2, ColabFold). Input sequences typically range from a few KB for single proteins to several MB for large datasets, and reference databases can be extremely large, from 100 GB to several TB. Protein folding pipelines generate intermediate files during execution, such as for alignments and feature representations, the sizes of which vary based on the number of sequences and the complexity of the analysis.
+
+Given the data sizes and computational intensity, production pipelines perform best with GPUs and low-latency, high-throughput cloud storage file handling.
+
+### GPUs
```

Comment:
Yes, I set `toc_max_heading_level: 2` in the frontmatter, to prevent the TOC from getting too long. I'd like to do the same for some other long pages that have a huge RHS TOC. Perhaps we should standardize on `toc_max_heading_level: 3` as a good middle-ground.